### DISCUSSION TITLE
Translate the site's title

### DISCUSSION THREAD

**Discussion by zkvvoob** [withastro/starlight#1557] [Feature Requests]
### What version of `starlight` are you using?

0.20.1

### What is your idea?

Would it be possible to localise/translate a site's title in the `astro.config.mjs` file, much like we can currently translate the sidebar items' labels?

### Why is this feature necessary?

Because keeping a Bulgarian title when switching to English makes little sense and is not particularly helpful to users.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Starlight [`0.22.0`](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.22.0) added support for translating the site title.

Documentation for this feature is available [here](https://starlight.astro.build/guides/i18n/#translate-the-site-title) but the change would be:

```diff
import { defineConfig } from 'astro/config';
import starlight from '@astrojs/starlight';

export default defineConfig({
  integrations: [
    starlight({
-      title: 'My Docs',
+      title: {
+         en: 'My Docs',
+        'zh-CN': '我的文档',
+     },
      defaultLocale: 'en',
      locales: {
        en: { label: 'English' },
        'zh-cn': { label: '简体中文', lang: 'zh-CN' },
      },
    }),
  ],
});
```

**Reply by zkvvoob** [reply]
Much obliged for implementing this, @HiDeoo!
